javacomparator/FileComparator.java

```diff
@@ -0,0 +1,186 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+
+public class FileComparator {
+    String fileOriginalPath, fileRoundtrippedPath;
+    String fileExtension;
+
+    FileComparator(String Path1, String Path2) {
+        fileOriginalPath = Path1;
```

Comment:
Again: please have better variable names which reflect what they are referring to.